kiwi/debug.h

```diff
@@ -180,7 +181,15 @@ namespace debug
 template<typename T>
 void dump( const T& value )
 {
-	impl::DebugHelper::dump( value );
+	impl::DebugHelper::dump( value, std::cout );
```

Comment:
Can you make an overload for `dump` which takes an `ostream` reference? That may be useful for people.